.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "ignorePatterns": [
+        // we probably want to change it to a function component,
+        // the error was:
+        // Parsing error: Unexpected token =
+        "components/unlock/unlock.js"
```

Comment:
I may change that to a functional component component later, but we may want to snapshot test it first